docs/src/dev-manual/devel.md

```diff
@@ -247,3 +258,4 @@ is sometimes not setup properly. To solve it, from the Tilt UI, restart the
 [gcc]: https://gcc.gnu.org/
 [visual studio code]: https://code.visualstudio.com/
 [working with archivematica]: archivematica.md
+[devbox]: https://www.jetpack.io/devbox/docs/quickstart/#install-devbox
```

Comment:
```suggestion
[Devbox]: https://www.jetpack.io/devbox/docs/quickstart/#install-devbox
```

Reply:
hmm this seems out of alighnment with how we have capitalised every other variable here.

Reply:
Oh maybe it's case insensitive? That makes sense. I only meant that the proper name is Devbox as opposed to devbox.